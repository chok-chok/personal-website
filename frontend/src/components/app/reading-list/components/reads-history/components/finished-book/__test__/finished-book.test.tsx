import * as React from 'react';
import FinishedBook from '../finished-book';
import { shallow } from 'enzyme';

describe('Test FinishedBook component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<FinishedBook />);

    expect(tree).toMatchSnapshot();
  });
});
