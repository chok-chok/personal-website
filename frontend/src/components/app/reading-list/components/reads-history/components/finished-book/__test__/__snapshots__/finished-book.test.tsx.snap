// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test FinishedBook component should renders correctly 1`] = `
<styled__FinishedBookBackground>
  <styled__FinishedBookImg
    src="/img/example-book-img.jpg"
  />
  <styled__FinishedBookDetailWrapper>
    <styled__FinishedBookLink
      href="https://www.amazon.com"
      target="_blank"
    >
      <styled__BookTitle>
        Domain Driven Design: Tackling Complexity in the Heart of Software
      </styled__BookTitle>
    </styled__FinishedBookLink>
    <styled__AuthorInfo>
      by Eric Evans
    </styled__AuthorInfo>
    <styled__RatingSection>
      My rating:
    </styled__RatingSection>
    <styled__ReadDate>
      Read date: 2020.10
    </styled__ReadDate>
    <styled__MyReview>
      Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. v Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book. Here is my reveiw about this book.
    </styled__MyReview>
  </styled__FinishedBookDetailWrapper>
</styled__FinishedBookBackground>
`;
